model.py

```diff
@@ -41,18 +42,29 @@ def update(student_Id, studentInfo):
     @staticmethod
     def deleteAll():
         dao=Dao()
-        try:
-          dao.deleteAll()
-          return 
-        except:
-          return
+        dao.deleteAll()
+        return
         
     @staticmethod
-    def deleteStudent(student_Id):
+    def delete(student_Id):
         dao = Dao()
         try:
           deleted=dao.delete(student_Id)
           return Model(deleted)
         except:    
           return None
 
+    @staticmethod
+    def getStudents():
+        dao = Dao()
+        allstud = dao.getStudents()
+        students = []
+        for s in allstud.find():
+            print (s,file=sys.stderr)
+            students.append(repr(Model(s)))
```

Comment:
```py
students.append(Model(s))
```

The model knows what the DAO is returning, if the `allstud` is not None .. it holds a list of json dictionaries